cli/cmd/install.go

```diff
@@ -65,6 +65,29 @@ func newInstallTarantoolEeCmd() *cobra.Command {
 	return tntCmd
 }
 
+// newInstallTarantoolDevCmd creates a command to install tarantool
+// from the local build directory.
+func newInstallTarantoolDevCmd() *cobra.Command {
+	tntCmd := &cobra.Command{
+		Use:   "tarantool-dev dir",
+		Short: "Install tarantool from the local build directory",
+		Example: "Assume, tarantool build directory is ~/src/tarantool/build\n" +
+			"  Consider the following use case:\n\n" +
+			"  tt install tarantool-dev ~/src/tarantool/build\n" +
```

Comment:
Will that scenario work? 
As i understand, compiled binary should be already placed at ~/src/tarantool/build.
Like :
 1. build binary
 2. tt install tarantool-dev

Reply:
Yes, you're right, it was implied. Now the sample has been changed to be more clear.